dracut/30ignition/ignition-check-neednet.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+set -euo pipefail
+
+set +euo pipefail
+. /usr/lib/dracut-lib.sh
+set -euo pipefail
+
+dracut_func() {
+    # dracut is not friendly to set -eu
+    set +euo pipefail
+    "$@"; rc=$?
```

Comment:
`local rc`

Reply:
Thanks, fixed! As well as the other instance this was based on.